packages/suite-web/e2e/cypress.config.ts

```diff
@@ -69,7 +69,7 @@ export default defineConfig({
         video: true,
         trashAssetsBeforeRuns: false,
         chromeWebSecurity: false,
-        experimentalFetchPolyfill: true,
+        // experimentalFetchPolyfill: true, // This makes Cardano webassembly loading fail.
```

Comment:
all the changes in this commit were required because of this going away?

Reply:
Nop, all the changes in this commit were fixing other issues before this one. No changes in the tests were required after removing this.
I am thinking that it will be better to delete the line `// experimentalFetchPolyfill:` instead of commenting.
It is also a deprecated experimental feature by cypress https://docs.cypress.io/app/references/experiments

Reply:
I am totally in favor of removing it. But also - won't it require bigger refactoring? They say you cant use cy.stub and need to use cy.intercept. We use the former many times in the code base at the moment 🤔

Reply:
Exactly but we use `cy.intercept` already. It is possible that config was never needed just added as a boilerplate. And tests are working properly without it.